src/main/webapp/script.js

```diff
@@ -352,89 +351,69 @@ function getDailyView(aggregated) {
     data.push(totalCostsOrdered);
   }
 
-  data = transpose(data);
+  for (var i = 1; i < 4; i++) {
+    var totalCost = data[1].slice(i, data[1].length).reduce((a, b) => a + b, 0);
+    totalCost /= 30;
```

Comment:
Is 30 coming from the total numbers of days in a month? Wouldn't be more accurate to use the actual number of days a certain month has?

Reply:
I chose thirty as originally we only had 30 days worth of data (because of the deleting issue) but also I think 30 days is a good number of days to display for the cost prediction - unless you perhaps think I should show more?